components/console_cmd_ping/README.md

```diff
@@ -0,0 +1,36 @@
+# Console command ping
+The component provides a console where the 'ping' command can be executed.
+
+## API
+
+### Steps to enable console in an example code:
+1. Add this component to your project using ```idf.py add-dependency``` command.
+2. In the main file of the example, add the following line:
+    ```c
+    #include "console_ping.h"
+    ```
+3. Ensure esp-netif is initialized and default event loop is created in your app_main():
+    ```c
+    ESP_ERROR_CHECK(esp_netif_init());
+    ESP_ERROR_CHECK(esp_event_loop_create_default());
+    ```
+4. In your app_main() function, add the following line as the last line:
+    ```c
+    console_cmd_init();     // Initialize console
+    console_cmd_ping_register();
+    console_cmd_start();    // Start console
+    ```
+
+
+## Suported commands:
+
+### ping:
+ping  [-W \<t>] [-i \<t>] [-s \<n>] [-c \<n>] [-Q \<n>] [-T \<n>] \<host><br>
+    send ICMP ECHO_REQUEST to network hosts<br>
+    -W, --timeout=\<t>  Time to wait for a response, in seconds<br>
+    -i, --interval=\<t>  Wait interval seconds between sending each packet<br>
+    -s, --size=\<n>  Specify the number of data bytes to be sent<br>
+    -c, --count=\<n>  Stop after sending count packets<br>
+    -Q, --tos=\<n>  Set Type of Service related bits in IP datagrams<br>
+    -T, --ttl=\<n>  Set Time to Live related bits in IP datagrams<br>
+        \<host>  Host address
```

Comment:
Looks like this part can be in a code block for nicer formatting: https://github.com/espressif/esp-protocols/blob/e71b04f8640c477e937c8d8fdf328f58e341791e/components/console_cmd_ping/README.md